assets/styles/global.css

```diff
@@ -31,3 +31,8 @@ select,
 textarea {
   font-family: inherit;
 }
+
+img {
+  max-width: 100% !important;
```

Comment:
is it really needed?
Using tag selector is not a very good idea anyway if it's not a reset CSS
And setting up `!important` will make it very hard to change which could affect different parts of code which will be not very easy to debug.
Tl;dr; this is an antipattern, please try to reconsider the usage of this approach

Reply:
Not, it's not.
I had 2 css files for some reason
I was trying to understand why css doesn't work and just forgot to remove this.

Reply:
👍